### DISCUSSION TITLE
Cost-basis pre June 2020 (US Taxes)

### DISCUSSION THREAD

**Discussion by mattsilv** [davetapley/helium-tax#10] [General]
This tool is really awesome.  Just curious if anyone has spoken to an account about how to treat the cost basis prior to June 2020.  I wonder if cost-basis could just be treated as 0 so there is no impact until those mined HNT are sold.

**Comment by davetapley** [comment]
I haven't spoke to an accountant, but I did find this which may be of interest:

> Q26.  I received cryptocurrency in a peer-to-peer transaction or some other type of transaction that did not involve a cryptocurrency exchange.  How do I determine the cryptocurrency’s fair market value at the time of receipt?
 > 
> A26.  If you receive cryptocurrency in a peer-to-peer transaction or some other transaction not facilitated by a cryptocurrency exchange, the fair market value of the cryptocurrency is determined as of the date and time the transaction is recorded on the distributed ledger, or would have been recorded on the ledger if it had been an on-chain transaction.  The IRS will accept as evidence of fair market value the value as determined by a cryptocurrency or blockchain explorer that analyzes worldwide indices of a cryptocurrency and calculates the value of the cryptocurrency at an exact date and time.  If you do not use an explorer value, you must establish that the value you used is an accurate representation of the cryptocurrency’s fair market value. 


https://www.irs.gov/individuals/international-taxpayers/frequently-asked-questions-on-virtual-currency-transactions

---

As for other similar apps:

1. https://helium.hiker.rocks/ is omitting it
2. https://www.minertax.com/HNT/usd/ is [using Coingecko](https://www.reddit.com/r/HeliumNetwork/comments/lgwn2y/helium_hotspot_tax_record_for_50_currencies/gmupqe4/?utm_source=reddit&utm_medium=web2x&context=3)
